Floofbot/Modules/Logging.cs

```diff
@@ -26,7 +27,7 @@ public LoggerCommands(FloofDataContext floofDB)
                 _floofDB = floofDB;
             }
 
-            protected void CheckServer(ulong server)
+            private void CheckServer(ulong server)

```

Comment:
What is the reason for changing these to private?

Reply:
Usually, you want to set things to the lowest privilege. The logic I use is as follows:
If I give the code a quick glance and I see private, I know it's only being called within this class. If I see protected, I'll have to start worrying about where else it's being called.
If I have to give more access later, then I can do it later.